aircraft/solar_uav.py

```diff
@@ -0,0 +1,62 @@
+"""LLhale by whoburg, inspired by Drela"""
+from gpkit import Variable, Model
+
+
+def solar_uav(substitutions=None):
+    """Returns the Model"""
+    # Performance
+    P = Variable("P", units="W", label="flight power")
+    eta_p = Variable(r"\eta_p", value=0.75, label="motor+prop efficiency")
+    L15D = Variable(r"(C_L^{1.5}/C_D)", value=25, label="power parameter")
+    rho = Variable(r"\rho", units="kg/m^3", label="air density",
+                   value=('sweep', [0.238, 0.449]))
+    wl = Variable("(m/S)", units="kg/m^2", label="wing loading")
+    g = Variable("g", units="m/s^2", value=9.8)
+
+    # solar cells
+    Psolar = Variable("P_{solar}", units="W/m^2", value=200,
+                      label="solar cell output")
```

Comment:
Ah, instead of keyword args? That's a fair point; I think I've been using explicit kwargs for a couple not-too-defendable reasons:

 - In case the arg order ever changes (and because I sometimes forget the order)
 - For readability for new users reading the model

I'm game to switch over to using an arg list, especially if you have a convincing reason to.